assembly/assembly-wsmaster-war/src/main/java/org/eclipse/che/api/deploy/MetricsOverrideBinding.java

```diff
@@ -0,0 +1,112 @@
+/*
+ * Copyright (c) 2012-2018 Red Hat, Inc.
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *   Red Hat, Inc. - initial API and implementation
+ */
+package org.eclipse.che.api.deploy;
+
+import com.google.inject.Binder;
+import com.google.inject.Module;
+import io.micrometer.core.instrument.Tags;
+import io.micrometer.prometheus.PrometheusMeterRegistry;
+import java.util.concurrent.ExecutorService;
+import javax.inject.Inject;
+import javax.inject.Named;
+import javax.inject.Singleton;
+import org.eclipse.che.api.deploy.jsonrpc.CheMajorWebSocketEndpointConfiguration;
+import org.eclipse.che.api.deploy.jsonrpc.CheMajorWebSocketEndpointExecutorServiceProvider;
+import org.eclipse.che.api.deploy.jsonrpc.CheMinorWebSocketEndpointConfiguration;
+import org.eclipse.che.api.deploy.jsonrpc.CheMinorWebSocketEndpointExecutorServiceProvider;
+import org.eclipse.che.core.metrics.ExecutorServiceMetrics;
+
+/**
+ * {@link com.google.inject.Module} that provide alternative implementation of {@link
```

Comment:
> Generally, I think it is a usual practice to use FQNs in the links that are not imported in the source file otherwise.

I'm OK with that but here these classes are imported. 
Or do you mean that it's always better to use FQN?